debezium-api/src/main/java/io/debezium/errors/DebeziumException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.errors;
```

Comment:
I don't like the idea of such "technical" packages (it's ok for the annotations IMO, but I'd limit it otherwise). I'd suggest to move the engine exception to the engine package and the `DebeziumException` just to `io.debezium`.

Reply:
@gunnarmorling ok, both works for me. I've chosen this approach to follow Kafka API. Exceptions moved